commit/chainfee/validate_observation.go

```diff
@@ -50,5 +50,15 @@ func (p *processor) ValidateObservation(
 		}
 	}
 
+	for _, update := range obs.ChainFeeUpdates {
+		if update.ChainFee.ExecutionFeePriceUSD == nil || update.ChainFee.ExecutionFeePriceUSD.Cmp(zero) <= 0 {
+			return fmt.Errorf("nil or non-positive %s", "execution fee price")
+		}
+
+		if update.ChainFee.DataAvFeePriceUSD == nil || update.ChainFee.DataAvFeePriceUSD.Cmp(zero) < 0 {
```

Comment:
I believe this DataAvFeePriceUSD might be zero for EVM chain and shouldn't be zero for L2 chains. Is that right? If at all we can know the given chain is L2, then we can add additional check here.